internal/agent/f5/endpoints_test.go

```diff
@@ -185,6 +188,8 @@ func TestAgent_ProcessEndpoint(t *testing.T) {
 	th.SetupPersistentPortHTTP(t, 8931)
 	defer th.TeardownHTTP()
 	config.Global.Agent.PhysicalNetwork = "physnet1"
+	config.Global.Agent.L4Profile = "/Common/cc_fastL4_profile"
+	config.Global.Agent.TCPProfile = "/Common/cc_tcp_profile"
```

Comment:
The desired profile name is `cc_tcp_archer_profile`, is this a typo or this is not referring to the new profile..?